src/lib/create-config.ts

```diff
@@ -324,10 +338,8 @@ const defaultAuthentication: IAuthOption = {
     type: authTypeFromString(process.env.AUTH_TYPE),
     customAuthHandler: defaultCustomAuthDenyAll,
     createAdminUser: true,
-    initialAdminUser: {
-        username: process.env.UNLEASH_DEFAULT_ADMIN_USERNAME ?? 'admin',
-        password: process.env.UNLEASH_DEFAULT_ADMIN_PASSWORD ?? 'unleash4all',
-    },
+    initialAdminUser: parseEnvVarInitialAdminUser(),
+    initialAdminEmailUser: parseEnvVarInitialAdminEmailUser(),
```

Comment:
Just a question: should it be one or the other? Just an idea:
```suggestion
    initialAdminUser: parseEnvVarInitialAdminEmailUser() ?? parseEnvVarInitialAdminUser(),
```

Then we can handle this complexity in the method. So an initial admin user is either a name + email or a username + password

The one benefit is that we have less fields in the config file, the second benefit is that we define the preference of email based over password based at an earlier stage.